run-tests-with-build-kogito-sources/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>run-tests-with-build</artifactId>
+    <groupId>org.kie</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <artifactId>run-tests-with-build-kogito-sources</artifactId>
+
+  <properties>
+    <maven.main.skip>false</maven.main.skip>
+    <maven.test.failure.ignore>false</maven.test.failure.ignore>
+  </properties>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.googlecode.maven-download-plugin</groupId>
+        <artifactId>download-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>prepare-sources-zip</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>wget</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <url>${kogito.sources.url}</url>
+          <unpack>true</unpack>
+          <outputDirectory>${sources.directory}</outputDirectory>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-invoker-plugin</artifactId>
+        <configuration>
+          <!-- kogito-runtimes tests were not designed to run in parallel -->
+          <parallelThreads>1</parallelThreads>
+          <profiles>productized</profiles>
+          <pomIncludes>
+            <pomInclude>kogito-*/*kogito-examples*/pom.xml</pomInclude>
+            <pomInclude>kogito-*/*kogito-runtimes*/pom.xml</pomInclude>
```

Comment:
How is that posssible we can run all runtimes and examples while in `run-tests-with-build-kogito-runtimes` and `run-tests-with-build-kogito-examples` we have exclusions?

Reply:
In case of examples and runtimes it's `productized` profile being passed a few lines above.

Reply:
To provide full justification here (based on off-line discussion). This new module does focus on building projects from sources.zip from top-level maven pom.xml only, does not focus on isolation of executions of respective sub-modules. This allows for using this profile.
In other cases we want to invoke explicitly lowest-level pom.xml files (leaves in maven project structure tree) so that we isolate invocations of particular modules from the maven reactor (so that we use dependencies from nexus rather than freshly built ones from the reactor). That limits the profile-based approach in other modules (*-kogito-runtimes, *-kogito-apps, etc).